doc/Language/regexes.pod6

```diff
@@ -261,9 +261,16 @@ Examples of word characters:
 
 =end table
 
-Note that the character classes C«<same>», C«<wb>» and C«<ww>» are
-so called zero-width assertions, which do not really match a
-character.
+The predefined character classes in the leftmost column are all of the form
+C«<name>», a hint to the fact that they are implemented as built-in L<named
+regexes|language/regexes#Subrules>. As such they are subject to the usual
+capturing semantics. This means, inter alia, that these classes do not only
+match, but also capture, installing a correspondingly named L<named
+capture|/language/regexes#Named_captures> in the resulting L<Match
+object|/type/Match>.
+
```

Comment:
Please mention that preceding them with a . prevents capture.

Reply:
Done.